staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -752,7 +752,7 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 			w.Write(LEVEL_0, "Node:\t%s\n", pod.Spec.NodeName+"/"+pod.Status.HostIP)
 		}
 		if pod.Status.StartTime != nil {
-			w.Write(LEVEL_0, "Start Time:\t%s\n", pod.Status.StartTime.Time.Format(time.RFC1123Z))
+			w.Write(LEVEL_0, "Start Time:\t%s\n", pod.Status.StartTime.Time.UTC().Format(time.RFC1123Z))
```

Comment:
all kubectl changes are up to the kubectl maintainers to review.

Reply:
Thank you very much, if you can, can you assign to kubectl maintainers

Reply:
let's see SIG Architecture's take on this topic before we proceed with the user facing changes.